nibabel/tests/test_round_trip.py

```diff
@@ -108,23 +108,23 @@ def test_round_trip():
     iuint_types = [t for t in iuint_types if t in nifti_supported]
     f_types = [np.float32, np.float64]
     # Expanding standard deviations
-    for i, sd_10 in enumerate(sd_10s):
+    for sd_10 in sd_10s:
```

Comment:
`i`, `j`, and `k` are not required, but I believe they appear as “comments” to clarify the intent. Keep them?

Reply:
I don't think the `i`, `j`, `k` provide any additional context here. No need to re-add as comments.